src/backend/lib/queue.js

```diff
@@ -1,13 +1,16 @@
 const Bull = require('bull');
-const { Redis, redisUrl } = require('./redis');
+const url = require('url');
+const { Redis, redisUrl, redisPassword } = require('./redis');
 const { logger } = require('../utils/logger');
 
 /**
  * Shared redis connections for pub/sub, see:
  * https://github.com/OptimalBits/bull/blob/28a2b9aa444d028fc5192c9bbdc9bb5811e77b08/PATTERNS.md#reusing-redis-connections
  */
-const client = new Redis(redisUrl);
-const subscriber = new Redis(redisUrl);
+const { port, host } = url.parse(redisUrl, true);
```

Comment:
Move all this into the `redis.js` file.  We should process all the `env` variables in there, and either expose `redisUrl` like we do now, or a `redisOptions` or something, that people can use to pass to the `Redis()` constructor.

Reply:
Moved to `redis.js` and added `redisOptions` containing `port`, `host`, and `password` to `_module.exports_`.